DeviceAdministration/Web/WebApiControllers/TelemetryApiController.cs

```diff
@@ -205,7 +207,9 @@ public class TelemetryApiController : WebApiControllerBase
                     var resultsModel = new AlertHistoryResultsModel();
 
                     IEnumerable<AlertHistoryItemModel> data =
-                        await _alertsLogic.LoadLatestAlertHistoryAsync(currentTime.AddMinutes(-CautionAlertMaxMinutes));
+                        await _alertsLogic.LoadLatestAlertHistoryAsync(
+                            currentTime.Subtract(CautionAlertMaxDelta), 
+                            MaxHistoryItems);
```

Comment:
Will change `MaxHistoryItems` to `DISPLAYED_HISTORY_ITEMS` (and fix a merge issue).